### ISSUE TITLE
Saving a file for the first time does no longer propose the score's title as the file name inside the save-dialog

### ISSUE THREAD

**Issue by oMrSmith** [musescore/MuseScore#17142]
### Issue type

Other type of issue

### Bug description

See title.

### Steps to reproduce

1. Create a new Score.
2. Change the title.
3. Save the file.

You will always see:
![image](https://user-images.githubusercontent.com/78870770/230071995-ed390f97-7908-48dd-a238-1bcd84ca333e.png)

### MuseScore Version

OS: Windows 10 Version 2009, Arch.: x86_64, MuseScore version (64-bit): 4.1.0-230940304, revision: github-musescore-musescore-abb91a4

### Regression

Yes, this used to work in Musescore 3.x and now is broken

### Operating system

Windows 10

**Comment by bkunda** [comment]
As far as I can tell in MuseScore 3.6, this only worked if you tried to "Ctrl+S" save the file. If you tried to "Save as..." (via the file menu or using Ctrl+Shift+S) you'd also get "Untitled" in the filename field of the save dialog. 

So to the extent this is a regression, I'd say only a very trivial one. Addressing it properly would also mean considering how this should work for all possible instances of saving a file (including cloud saving). 

As a minor aside, I don't think it should always be assumed that the title of the work would necessarily correspond to the filename the user wishes to use. Of course the score title would _usually_ appear somewhere in the filename, but we could be splitting hairs here. 

I think we can slate this one for further consideration in the long-list of priorities for now.

**Comment by oMrSmith** [comment]
> As far as I can tell in MuseScore 3.6, this only worked if you tried to "Ctrl+S" save the file.

Just for the record I just tested: If you save for the first time, the score's title is proposed as the file for both actions "save" and "save as" in MU3. I think this makes sense as the score's title is more likely to become the filename as "untitled score" which also seems like a joke if you have just changed the title.

**Comment by bkunda** [comment]
Attaching my test record here. As far as I can tell, it seems that the behaviour is not consistent in MS3:

https://www.dropbox.com/s/gjdv6wtm5x91zm2/untitled-score-name-issue.mov?dl=0

**Comment by oMrSmith** [comment]
Yeah, it looks so. Also you're getting "untitled" wheras I see "untitled score". Maybe that's because it was taken out...
